src/gluonts/representation/representation_chain.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from .representation import Representation
+
+# Standard library imports
+import numpy as np
+from typing import Tuple, Optional, List
+import mxnet as mx
+
+# First-party imports
+from gluonts.core.component import validated, get_mxnet_context
+from gluonts.model.common import Tensor
+from gluonts.dataset.common import Dataset
+
+
+class RepresentationChain(Representation):
+    """
+    A class representing a hybrid approach of combining multiple representations into a single representation.
+    Representations will be combined by concatenating them on dim=-1.
+
+    Parameters
+    ----------
+    representations
```

Comment:
This was probably changed in the code:

```suggestion
    chain
```